app/models/provider.rb

```diff
@@ -46,6 +46,7 @@ class Provider < ApplicationRecord
 
   has_many :sites
   has_many :enrichments,
+           -> { where(status: 1).order(last_published_at: :desc) },
```

Comment:
```suggestion
           -> { where(status: :published).order(last_published_at: :desc) },
```

Reply:
Use the scope for ordering - `latest_published_at`